### PR TITLE
Add `sandbox` to `securityLevel` docs

### PR DESCRIPTION
**Pull request by sidharthv96**
## :bookmark_tabs: Summary
- Add `sandbox` to `securityLevel` docs
- Fix prettier which was not running inside vscode
### :clipboard: Tasks
Make sure you
- [x] :book: have read the [contribution guidelines](https://github.com/mermaid-js/mermaid/blob/develop/CONTRIBUTING.md) 
- [x] :computer: have added unit/e2e tests (if appropriate) 
- [x] :bookmark: targeted `develop` branch

### CODE REVIEW

File: docs/Setup.md [hunk 1]
```diff
@@ -1494,10 +1494,10 @@ $(function () {
 -   `_txt` **any** The graph definition
 -   `cb` **any** Callback which is called after rendering is finished with the svg code as inparam.
 -   `container` **any** Selector to element in which a div with the graph temporarily will be
-      inserted. In one is provided a hidden div will be inserted in the body of the page instead. The
-      element will be removed when rendering is completed.
+    inserted. In one is provided a hidden div will be inserted in the body of the page instead. The
```
Comment (lorint) [inline docs/Setup.md:1497]: Should be "If one is provided..." instead of "In one is provided..."

File: docs/Setup.md [hunk 2]
```diff
@@ -72,16 +72,16 @@ Theme , the CSS style sheet
...
+| securityLevel | Level of trust for parsed diagram | string | Required | 'sandbox', 'strict', 'loose', 'antiscript' |
 
 **Notes**:
 
 -   **strict**: (**default**) tags in text are encoded, click functionality is disabled
 -   **loose**: tags in text are allowed, click functionality is enabled
--   **antiscript**: html tags in text are allowed, (only script element is removed), click
-    functionality is enabled
+-   **antiscript**: html tags in text are allowed, (only script element is removed), click functionality is enabled
+-   **sandbox**: With this security level all rendering takes place in a sandboxed iframe. This prevent any JavaScript running in the context. This may hinder interactive functionality of the diagram like scripts, popups in sequence diagram or links to other tabs/targets etc.
```
Comment (lorint) [inline docs/Setup.md:84]: Second sentence would be more understandable as: "This prevents any JavaScript from running in the context."

**Review by lorint — COMMENTED** [review]
A couple ideas for better clarity

File: docs/Setup.md (hunk 1 above)
Comment (sidharthv96) [reply docs/Setup.md]: Thanks @lorint.

File: docs/Setup.md (hunk 2 above)
Comment (sidharthv96) [reply docs/Setup.md]: Fixed.

**Review by knsv — APPROVED** [review]

**Pull request merged**